_docs/hackathon-preperation-checklist.md

```diff
@@ -0,0 +1,43 @@
+# The Week Before
```

Comment:
Needs Jekyll headers here so that the page gets listed in the site menu.

````markdown
---
layout: default
category: Getting Started
title: Preparing for the Event
order: 3
permalink: /preparing.html
---
````